cachito/web/status.py

```diff
@@ -26,6 +26,7 @@
 }
 
 no_retry_session = requests.Session()
+service_timeout = 5
```

Comment:
Constants should generally be uppercase, there are some exceptions such as request sessions and logger objects but this should be uppercase IMO